addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -2605,7 +2604,6 @@ void handle_gameFrameLogic()
 							}
 
 							TF2_RemoveCondition(i, TFCond_Charging);
-							DealDamage(z, 1, i, _, "tf_wearable_demoshield");
```

Comment:
are you sure that you want to make the charge deal no damage?

Reply:
Damage is done here instead, replacing `SetEntityHealth`

https://github.com/redsunservers/SuperZombieFortress/pull/33/files#diff-8acb288cbd7093067921426ab032518aR2594

Reply:
i'm blind 😩